docs/source/getting_started/installation.rst

```diff
@@ -98,6 +98,38 @@ Usage with JupyterHub
 Read the details on our :ref:`JupyterLab on JupyterHub documentation page <jupyterhub>`.
 
 
+Usage with Jupyverse
+--------------------
+
+`Jupyverse <https://github.com/jupyter-server/jupyverse/>`__ is a next-generation Jupyter server based on
+`FastAPI <https://fastapi.tiangolo.com/>`__. It can be used instead of
+`jupyter-server <https://github.com/jupyter-server/jupyter_server/>`__, the Jupyter server installed by default with JupyterLab.
+Note that ``jupyter-server`` extensions won't work with ``jupyverse`` (for which there might be equivalent plugins).
+
+You can install ``jupyverse`` with ``pip``:
+
+.. code:: bash
+
+    pip install jupyverse[auth, jupyterlab]
+
+or with ``conda``:
+
+.. code:: bash
+
+    conda install -c conda-forge jupyverse fps-auth fps-jupyterlab
+
+or with ``mamba``:
+
+.. code:: bash
+
+    mamba install -c conda-forge jupyverse fps-auth fps-jupyterlab
+
+And run it with:
+
+.. code:: bash
+
+    jupyverse
```

Comment:
So will running `jupyverse` in the terminal run a *JupyterLab* session in my browser?

i.e. does `jupyverse` replace the `jupyter lab` command I use now? And, if so, can I pass all the same JupyterLab arguments to `jupyverse`?

Reply:
> So will running `jupyverse` in the terminal run a _JupyterLab_ session in my browser?

If jupyverse was installed with `pip install jupyverse[auth, jupyterlab]`, yes. If installed with `pip install jupyverse[auth, notebook]`, it will run Jupyter Notebook.

> i.e. does `jupyverse` replace the `jupyter lab` command I use now? And, if so, can I pass all the same JupyterLab arguments to `jupyverse`?

Yes, but arguments are different. More information is available in the [documentation](https://jupyter-server.github.io/jupyverse).